src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -0,0 +1,397 @@
+package org.wildfly.security.http.impl;
```

Comment:
Sorry @honza889 I may have missed this before but this one is missing a copyright header.